src/GafferSceneUI/SceneGadget.cpp

```diff
@@ -780,6 +784,51 @@ Imath::Box3f SceneGadget::bound() const
 	return bound( /* selection = */ false );
 }
 
+void SceneGadget::snapshotToFile(
+	const std::filesystem::path &fileName,
+	const Box2f &resolutionGate,
+	const CompoundData *metadata
+) const
+{
+	if( !m_outputBuffer )
+	{
+		return;
+	}
+
+	std::filesystem::create_directories( fileName.parent_path() );
+
+	const Box2i &dataWindow = m_outputBuffer->dataWindow();
+	OIIO::ImageSpec spec( dataWindow.size().x + 1, dataWindow.size().y + 1, 4, OIIO::TypeDesc::FLOAT );
+
+	if( !resolutionGate.isEmpty() )
+	{
+		spec.x = -resolutionGate.min.x;
+		spec.y = -resolutionGate.min.y;
+
+		spec.full_x = 0;
+		spec.full_y = 0;
+		spec.full_width = resolutionGate.size().x;
+		spec.full_height = resolutionGate.size().y;
+	}
+
+	const std::vector<float> &rgbaBuffer = m_outputBuffer->rgbaBuffer();
+
+	const IECoreImage::OpenImageIOAlgo::DataView dataView( metadata );
```

Comment:
This `dataView` is unused as far as I can see, and shadowed by the one inside the loop.

Reply:
Fixed amongst https://github.com/GafferHQ/gaffer/pull/5533/commits/b6c2f5367845b1fe6f4d802a0ea795e2c95c9fa6 and https://github.com/GafferHQ/gaffer/pull/5533/commits/5dead3ccef2b62c176f6f43e542c7f16794a6df6